lib/m2ts/m2ts.js

```diff
@@ -254,7 +254,7 @@ TransportParseStream.STREAM_TYPES  = {
  * packets plus relevant metadata that has been extracted from the
  * container.
  */
-ElementaryStream = function() {
+ElementaryStream = function(timelineBaseDts) {
```

Comment:
Naming nitpick: I think we should stick with MSE terminology where it makes sense and this seems like a good candidate. `timelineOffset`?